internal/applicationsnapshot/report.go

```diff
@@ -254,16 +258,47 @@ func (r *Report) toAppstudioReport() testReport {
 		}
 	}
 
+	result.deriveResult(hasFailures)
+	return result
+}
+
+// Used in cmd/test/test
+func AppstudioReportFromCheckResults(results []conftestOutput.CheckResult, namespaces []string) testReport {
+	// This is may need revising in future. It does not handle multiple namespaces
+	// accurately. We could consider using a string delimited list of namespaces
+	// but I'm being cautious about breaking consumers of this data. The
+	// testReport.Namespace field might need to be converted to a list of strings
+	// in future but we need to coordinate that change carefully.
+	// Also, we might prefer to extract the namespaces from results rather than
+	// use whatever the user provided on the command line.
+	useNamespace := ""
+	if len(namespaces) > 0 {
+		// The first namespace only
+		useNamespace = namespaces[0]
+	}
+	report := testReport{
+		Timestamp: fmt.Sprint(time.Now().UTC().Unix()),
+		Namespace: useNamespace,
+	}
+
+	for _, result := range results {
```

Comment:
Do we need to handle skipped here?

Reply:
(After some slack discussion) I did add extra test coverage for result "SKIPPED". Might need some more work in future around Conftest exceptions.